### PR TITLE
feat: add dynamic config for slippage

### PR DESCRIPTION
**Pull request by kathaypacific**
### Description

This PR adds plumbing to show the allowed slippage. Our swap endpoint accepts a slippagePercentage query param already. and is defaulting to 0.3% if not provided. Since we now need this on the client, it made sense to me to control this remotely.

The dynamic config setup is [here](https://console.statsig.com/4plizaPmWwPL21ASV4QAO0/dynamic_configs/swap_config).

### Test plan

Tested that we can still get swap quotes as usual.

### Related issues

Related to RET-865

### Backwards compatibility

Y

### CODE REVIEW

File: src/statsig/constants.ts
Comment (jeanregisser) [inline src/statsig/constants.ts:116]:
```suggestion
      maxSlippagePercentage: '0.3',
```

**Review by jeanregisser — APPROVED** [review]
👍

**Review by bakoushin — APPROVED** [review]
LGTM!

I took me some time to realize that this PR is _not_ into the `main` branch 😅 
Is that even legal?

**Comment by kathaypacific** [comment]
> LGTM!
> 
> I took me some time to realize that this PR is _not_ into the `main` branch 😅 Is that even legal?

haha yes i do this quite frequently...although i never merge into the target branch, i'd usually wait for the target branch to be merged to main first before merging these PRs. i think it helps to keep PRs smaller and means i can keep working on stuff while waiting for reviews :)

**Pull request merged**
